server.go

```diff
@@ -222,6 +222,14 @@ func NewFromConfig(conf Config) (ret Server, err error) {
 
 	ret.EventWorker = NewEventWorker(ret.Statsd)
 
+	// Set up a span sink that extracts metrics from SSF spans and
+	// reports them via the metric workers:
+	metricSink, err := NewMetricExtractionSink(ret.Workers, ret.indicatorSpanTimerName)
```

Comment:
We should maybe make this configurable? (I say maybe cuz this *is* kinda the core thing we aim to eventually do here, but in the short and medium term it might be chunderous for some installs?)

Reply:
Pretty against having this configurable: when we remove the statsd stuff in favor of the internal client, and you configure this off, you have lost all veneur metrics.

Reply:
Ok. I guess it *is* our value proposition. :)